openpype/hosts/maya/plugins/publish/validate_rig_contents.py

```diff
@@ -148,3 +151,63 @@ def validate_controls(self, set_members):
                 invalid.append(node)
 
         return invalid
+
+    def validate_skeleton_sets(self, instance):
+        objectsets = ["skeletonAnim_SET", "skeletonMesh_SET"]
+        missing = [obj for obj in objectsets if obj not in instance]
+        if missing:
+            self.log.debug("%s is missing %s" % (instance, missing))
```

Comment:
Are these optional or required? I believe they are optional. Right?
Also, should we put this validation for skeleton rig sets into a validator of its own? It's a relatively specific validator. It's best to have separate dedicated validators so we can also create the best consistent client facing reports for each validation.

```suggestion
        required = ["skeletonAnim_SET", "skeletonMesh_SET"]
        missing = [
            key for key in required if key not in instance.data["rig_sets"]
        ]
        if missing:
            self.log.debug("%s is missing %s" % (instance, missing))
            return
```